Bug-Ubuntu: https://bugs.launchpad.net/bugs/1986852
Origin: backport, https://git.savannah.nongnu.org/gitweb/?p=dmidecode.git;a=commit;h=3882a45fb6b7a36ba9463ee53e46a9090ecb8d8c
From: Jean Delvare <jdelvare@suse.de>
Date: Wed, 15 Jun 2022 13:35:25 +0200
Subject: [PATCH] dmidecode: Set supported version to 3.5.0

While we do not support every piece of the specification yet, the
most needed ones are in place now so it seems reasonable to stop
warning users.

Signed-off-by: Jean Delvare <jdelvare@suse.de>
[mfo: backport: refresh upper context lines.]
---
 dmidecode.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dmidecode.c b/dmidecode.c
index d93cd6883a67..9aeff91a71bd 100644
--- a/dmidecode.c
+++ b/dmidecode.c
@@ -87,7 +87,7 @@ static const char *bad_index = "<BAD INDEX>";
 #define out_of_spec "<OUT OF SPEC>"
 static const char *bad_index = "<BAD INDEX>";
 
-#define SUPPORTED_SMBIOS_VER 0x030300
+#define SUPPORTED_SMBIOS_VER 0x030500
 
 #define FLAG_NO_FILE_OFFSET     (1 << 0)
 #define FLAG_STOP_AT_EOT        (1 << 1)
-- 
2.34.1

